app/code/community/Oyst/OneClick/Model/Catalog.php

```diff
@@ -32,7 +32,7 @@ class Oyst_OneClick_Model_Catalog extends Mage_Core_Model_Abstract
         Mage_Catalog_Model_Product_Type::TYPE_CONFIGURABLE,
         Mage_Catalog_Model_Product_Type::TYPE_GROUPED,
         //Mage_Catalog_Model_Product_Type::TYPE_BUNDLE,
-        //Mage_Catalog_Model_Product_Type::TYPE_VIRTUAL,
+        Mage_Catalog_Model_Product_Type::TYPE_VIRTUAL,
         //Mage_Downloadable_Model_Product_Type::TYPE_DOWNLOADABLE,
```

Comment:
As I know of the Oyst thesaurus TYPE_VIRTUAL and TYPE_DOWNLOADABLE are managed similarly.